### PR TITLE
Resolve results bug and move loading conditional within app to results component

### PR DESCRIPTION
**Pull request by priscillaapaxton**
Results are now reset when user clicks on dachshund return home logo & conditional rendering based on loading property within state has been moved to results component 

# PR Template

### Types of changes made?

- [ ]  testing
- [ ]  documentation
- [x]  javascript
- [ ]  functionality

### What does this PR do?

- Adds functionality
- Updates JS for conditional rendering based on loading/error components
- Fixes bug for resetting results array within app's state
- Writes test for

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -45,10 +45,16 @@ class App extends Component {
...
+    this.setState({
+      results: this.state.places
+    })
+  };
+
   render() {
     return (
       <>
-        <Header />
+        <Header resetResults={this.resetResults}/>
```
Comment (tialaaa) [inline src/App/App.js:57]: Great solution to the bug we all discussed!! 👏👏👏

File: src/App/App.js [hunk 2]
```diff
@@ -58,11 +64,7 @@ class App extends Component {
                 <>
                   <Search filterPlaces={this.filterPlaces} />
                   <div className='results-component-container'>
-                    {this.state.loading ? (
-                    <p className='loading-message'>Loading your favorite places...</p> 
-                    ) : ( 
-                      <Results places={this.state.results} /> 
-                    )}
+                      <Results places={this.state.results} loading={this.state.loading}/>
```
Comment (tialaaa) [inline src/App/App.js:67]: Great refactoring here, too!

**Review by Pma913 — COMMENTED** [review]
Nice work implementing the reset function to the logo 👍

**Comment by hjawad22** [comment]
Looks great, thanks for these updates!

**Pull request merged**